spec/webauthn/attestation_statement/android_safetynet_spec.rb

```diff
@@ -84,15 +85,31 @@
       let(:nonce) { Base64.strict_encode64(OpenSSL::Digest::SHA256.digest("something else")) }
 
       it "returns false" do
-        expect(statement.valid?(authenticator_data, client_data_hash)).to be_falsy
+        expect(statement.valid?(authenticator_data, client_data_hash, trust_store: trust_store)).to be_falsy
       end
     end
 
     context "when ctsProfileMatch is not true" do
       let(:cts_profile_match) { false }
 
       it "returns false" do
-        expect(statement.valid?(authenticator_data, client_data_hash)).to be_falsy
+        expect(statement.valid?(authenticator_data, client_data_hash, trust_store: trust_store)).to be_falsy
+      end
+    end
+
+    context "when timestampMs is set to future" do
+      let(:timestamp) { (Time.now.to_i + 60) * 1000 }
+
+      it "returns false" do
+        expect(statement.valid?(authenticator_data, client_data_hash, trust_store: trust_store)).to be_falsy
+      end
+    end
+
+    context "when timestampMs is older than a minute old" do
```

Comment:
"older than one minute" or "one minute old"?

Reply:
It's testing "one minute old" but it should be "older than one minute", like the conformance test. Will fix it!